src/dkg/centralized_party/commitment_round.rs

```diff
@@ -78,6 +78,9 @@ impl<
         ProtocolContext,
     >
 {
+    /// This function implements Protocol 4, step 1 of the
+    /// 2PC-MPC: Emulating Two Party ECDSA in Large-Scale MPC paper.
+    /// src: https://eprint.iacr.org/2024/253
```

Comment:
```suggestion
    /// src: https://eprint.iacr.org/archive/2024/253/20240217:153208
```
ePrint keeps track of the version of submission, the numbering might change.

Reply:
Applied this across the board.